src/search/QuickSearchField.js

```diff
@@ -91,6 +91,13 @@ define(function (require, exports, module) {
 
         $input.on("input", this._handleInput);
         $input.on("keydown", this._handleKeyDown);
+        
+        // For search History this flag is set to false
+        if (options.shouldHighLightFirstIndex === undefined) {
+            this._shouldHighLightFirstIndex = true;
```

Comment:
In this case if we pass false then it will taken as true, that's why I had to explicitly add the conditional statements